ansible_collections/arista/avd/docs/plugins/Modules_and_action_plugins/eos_validate_state_runner.md

```diff
@@ -42,7 +42,7 @@ The plugin offers the following capabilities:
 | <samp>custom_anta_catalogs_dir</samp> | any | optional | None |  | The directory where custom ANTA test catalogs are stored.<br>Files must be named after the device hostname or the Ansible group name and have a `.yml` or `.yaml` extension. |
 | <samp>skipped_tests</samp> | list | optional | None |  | A list of dictionaries specifying categories and, optionally, tests to skip.<br>Each dictionary must have a key `category` and can optionally include a `tests` key. |
 | <samp>&nbsp;&nbsp;&nbsp;&nbsp;category</samp> | str | optional | None |  | The name of an AvdTest category (e.g., `AvdTestHardware`). |
-| <samp>&nbsp;&nbsp;&nbsp;&nbsp;tests</samp> | list | optional | None |  | An optional list of specific tests in the category to skip (e.g., `VerifyRoutingProtocolModel` in `AvdTestBGP`).<br>If not specified, all tests in the category are considered.<br>For a complete list of available tests, see [link to the test list](https://avd.sh/en/stable/roles/eos_validate_state/ANTA-Preview.html#test-categories). |
+| <samp>&nbsp;&nbsp;&nbsp;&nbsp;tests</samp> | list | optional | None |  | An optional list of specific tests in the category to skip (e.g., `VerifyRoutingProtocolModel` in `AvdTestBGP`).<br>If not specified, all tests in the category are considered.<br>For a complete list of available tests, see [link to the test list](https://avd.sh/en/stable/roles/eos_validate_state/anta_integration.html#test-categories). |
```

Comment:
In this file at line 58 and there is eos_validate_state_reports.md on same location where still shows the ninja link of anta.
https://github.com/aristanetworks/avd/blob/devel/ansible_collections/arista/avd/docs/plugins/Modules_and_action_plugins/eos_validate_state_reports.md#:~:text=ANTA%20website%3A%20https%3A//anta.ninja

need to update it on below path
ansible-avd/ansible_collections/arista/avd/plugins/modules/eos_validate_state_reports.py
ansible-avd/ansible_collections/arista/avd/plugins/modules/eos_validate_state_runner.py